spyder/config/main.py

```diff
@@ -643,7 +643,6 @@
         ('run', [
             'breakpoints',
             'configurations',
-            'defaultconfiguration',
```

Comment:
There is no need to bump the config version, right?

Reply:
This config option wasn't used anywhere. That's why I didn't bump `CONF_VERSION`.